subprojects/performance/src/performanceTest/groovy/org/gradle/performance/regression/android/FasterIncrementalAndroidBuildsPerformanceTest.groovy

```diff
@@ -111,6 +111,7 @@ class FasterIncrementalAndroidBuildsPerformanceTest extends AbstractCrossBuildPe
             builder.invocation.args(AndroidGradlePluginVersions.OVERRIDE_VERSION_CHECK)
             builder.invocation.args("-Dorg.gradle.workers.max=8", "--no-build-cache", "--no-scan")
             builder.invocation.useToolingApi()
+            builder.invocationCount(100)
```

Comment:
I guess we could roll this down to something like 60. WDYT?

BTW, are we down to 1 warmups here, too?

Reply:
I now use 1 warmup and 60 iterations.